source/includes/api/catalog/_brand.md

```diff
@@ -124,43 +94,24 @@ curl -X "GET" "https://dashboard.shoprunback.com/api/v1/brands" \
 
 ```php
 <?php
+// Load the library
+require 'path/to/lib/shoprunback-php/init.php';
 
-// Get cURL resource
-$ch = curl_init();
-
-// Set url
-curl_setopt($ch, CURLOPT_URL, 'https://dashboard.shoprunback.com/api/v1/brands');
-
-// Set method
-curl_setopt($ch, CURLOPT_CUSTOMREQUEST, 'GET');
+// Set your token
+\Shoprunback\RestClient::getClient()->setToken('your_token');
 
-// Set options
-curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
+// Get all your brands
+$brands = \Shoprunback\Elements\Brand::all();
+$brands[0]; // Returns the Brand with the index 0
```

Comment:
we dont need this, the endpoint is to List all the brands, not to get index[0] or index[1]
you can remove the code after `$brands = \Shoprunback\Elements\Brand::all();`